tasks/main.yml

```diff
@@ -44,6 +44,16 @@
   notify:
     - restart keepalived
 
+- name: install keepalived scripts dependencies
```

Comment:
Can you add those to the module which installs the package, just above? No need to go twice through package manager.

Thank you!

Reply:
Thanks for taking a look! The suggested change is done in b41410d

Reply:
My pleasure! Not sure how I missed the notifications... I will have to dig into that.